services/distribution/src/main/resources/master-config/v2/risk-calculation-parameters.yaml

```diff
@@ -13,6 +13,7 @@ minutes-at-attenuation-filters:
       min: 0
       max: 73
       max-exclusive: true
+  -
```

Comment:
@mlenkeit this additional `-` is correct, right?

Reply:
no, thanks for catching this!